common/components/WalletDecrypt/PrivateKey.jsx

```diff
@@ -80,7 +84,9 @@ export default class PrivateKeyDecrypt extends Component {
 
   onPasswordChange = (e: SyntheticInputEvent) => {
     const fixedPkey = fixPkey(this.props.value.key);
-    const isValid = isValidPrivKey(fixedPkey.length);
+    const isValid =
+      isValidPrivKey(fixedPkey.length) ||
+      isValidEncryptedPrivKey(fixedPkey.length);
```

Comment:
Created function `validatePkeyAndPass` to DRY things up :smile: how does that look @wbobeirne?